install/consul/istio.yaml

```diff
@@ -36,7 +38,7 @@ services:
           - consul
     ports:
       - "8500:8500"
-      - "172.28.0.1:53:8600/udp"
+      - "53:8600/udp"
       - "8400:8400"
```

Comment:
I think we need "53:8600/udp" for Mac but "172.28.0.1:53:8600/udp" for linux, due to docker network differences.  @GregHanson can you confirm that?

Reply:
Yes, I spoke to @GregHanson . He is adding two YAMLs (one for mac and one for linux)